src/components/Footer/FooterSocial/FooterSocialItem.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import cx from 'classnames';
+
+type FooterSocialItemProps = {
+  text: string;
+  url: string;
+};
+
+export const FooterSocialItem = ({ text, url }: FooterSocialItemProps) => {
+  return (
+    <a href={url} rel="noopener noreferrer" target="_blank">
+      <span className="visuallyhidden">Wellcome on {text}</span>
```

Comment:
`visuallyhidden` should be `u-visually-hidden` (see https://github.com/wellcometrust/corporate-components/blob/develop/src/assets/styles/50-utilities/_a11y.scss)